src/rgw/rgw_admin.cc

```diff
@@ -395,6 +395,7 @@ void usage()
   generic_client_usage();
 }
 
+
```

Comment:
I like the minimalism.  Maybe we could make this an enum class while we're here?